functions/v2/index.js

```diff
@@ -14,34 +14,27 @@
 
 'use strict';
 
+const functions = require('@google-cloud/functions-framework');
+
 // [START functions_cloudevent_pubsub]
-/**
- * CloudEvent function to be triggered by Pub/Sub.
- * This function is exported by index.js, and executed when
- * the trigger topic receives a message.
- *
- * @param {object} cloudevent A CloudEvent containing the Pub/Sub message.
- * @param {object} cloudevent.data.message The Pub/Sub message itself.
- */
-exports.helloPubSub = cloudevent => {
+// Register a CloudEvent callback with the Functions Framework that will
+// be executed when the Pub/Sub trigger topic receives a message.
+functions.cloudEvent('helloPubSub', cloudevent => {
```

Comment:
Same comment as the last PR - instead of doing:
```
functions.cloudEvent('helloPubSub', ...)
```
can we do:
```
const helloPubSub = ...

// Expose the function
functions.cloudEvent('helloPubSub', helloPubSub)
exports.helloPubSub = helloPubSub
```

Reply:
I don't think this is what we want customers to use the API. There get better implicit typing and IDE autocompleting by passing a callback function directly to the `cloudEvent` function.

This in now unit testable using the helpers we released in v2.1.0.